### DISCUSSION TITLE
Prisma 1 demo db dump request

### DISCUSSION THREAD

**Discussion by TheoMer** [prisma/prisma#3563] [Q&A] [answered]
Can I possibly get a dump of my Prisma 1 db situated at https://eu1.prisma.sh?

**Comment by ryands17** [comment, marked as answer]
@TheoMer 
Could you share your email address and your workspace slug so that we can send you the dump of the data on your email address.

**Reply by TheoMer** [reply]
Hey @ryands17 my details are:

email: 6e240757@use.startmail.com
slug: TheoMer-1/sick-fits/dev

**Reply by ryands17** [reply]
Hey @TheoMer sent it on the above email address :)

**Reply by TheoMer** [reply]
Hey @ryands17 many thanks.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
